unit/testing-utils/require_class_not_stub.cpp

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************\
```

Comment:
If it's not too much of a pain, I'd be tempt to suggest you name this file (and it's corresponding header file, of course) to something more generic - e.g. `require_symbol.cpp/.h` so that in future other helper functions can be added. I suggest `require_symbol` only because the one utility function it currently contains takes a `symbolt` as an argument, but feel free to choose another name if you feel its more appropriate.

Reply:
Oh, and if you do rename the file, remember to update the #ifdef's in the header file to match the new file name...